packages/kbn-apm-synthtrace-client/src/lib/apm/instance.ts

```diff
@@ -75,7 +76,7 @@ export class Instance extends Entity<ApmFields> {
     return new ApmError({
       ...this.fields,
       'error.exception': [{ message, ...(type ? { type } : {}) }],
-      'error.grouping_name': groupingName || message,
+      'error.grouping_name': groupingName || sha256(message),
```

Comment:
Previously the default `groupingName`/`error.grouping_key` was the message. This would break links since `grouping_key` is expected to be an alphanumeric string.

Reply:
> Why do we even have grouping_name? 

Looks like it's never used so removing.

> Also, can we use fnv1a rather than sha256 for performance reasons?

Do you think it makes a difference in this case? Either way, happy to change if it's available. I only see it defined it profiling. Can I simply import from there?

Reply:
> Can I simply import from there?

Doesn't look like it so leaving for now.